openapi-bal-task-plugin/src/main/resources/openapi-options-schema.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "client-methods": {
+      "type": "string"
+    },
+    "operations": {
+      "type": "string"
+    },
+    "tags": {
+      "type": "string"
+    },
+    "nullable": {
+      "type": "string"
+    },
+    "mode": {
+      "type": "string"
```

Comment:
If we are supporting enum type and there is an automatic validation, it is better to have that right?

Reply:
Tried with the enum value and got and JSON parser issue. FTM shall we go with string type? I will improve this with the enum once the JSON error is fixed.